doc/source/user-environment.rst

```diff
@@ -33,6 +33,11 @@ Use an existing Docker image
    match the version installed by the helm chart that you're using. For example,
    ``v0.5`` of the helm chart uses  ``jupyterhub==0.8``.
 
+.. note::
+
+   You can find the configuration for the default Docker image used in this
+   guide `here <https://github.com/jupyterhub/zero-to-jupyterhub-k8s>`_.
```

Comment:
I'm a bit confused - there's no Dockerfile in the root of the z2jh repository...is that the right link?

Reply:
Might be looking for https://github.com/jupyterhub/zero-to-jupyterhub-k8s/blob/master/images/singleuser-sample/Dockerfile which is broken until https://github.com/jupyterhub/zero-to-jupyterhub-k8s/pull/439 is merged

Reply:
Indeed, pasting links while not paying attention 🚓

Reply:
I linked to the directory so people also get the README etc.

Reply:
We need to rewrite the README completely, since it's 100% out of date...